frontend/packages/@depmap/dataset-manager/src/components/index.tsx

```diff
@@ -47,8 +45,20 @@ export default function Datasets() {
     [key: string]: string;
   } | null>(null);
 
+  // TODO: Remove
```

Comment:
I think I see somethings from the other PR that I just approved. I'll ignore in this one.